Pearson-Advance/edx-platform
# PADV-377 - Update inputs in build and push workflows

## Ticket

https://agile-jira.pearson.com/browse/PADV-377

## Description

This PR changes some inputs as [tutor-build-image-action](https://github.com/Pearson-Advance/tutor-build-image-action/tree/vue/PADV-377) updated the names of its parameters.

## Type of Change

- [x] Change and add new input in workflow for prod.
- [x] Change and add new input in workflow for stage.

## How to test

- Create a test PR in the discovery repository.
- Merge this PR into the target branch.
- Check that the Github Action is triggered by the PR merge event.
- Check the output of the image build creation for any issues or silent issues.
-  When the process is complete, check the Docker Hub Pearson repository and verify that the image has been built (https://hub.docker.com/repository/docker/paops/edxapp/general).
-  Use the image in a local environment to test it out.

## Reviewers
